latest_only.sh

```diff
@@ -0,0 +1,7 @@
+	echo \
+		DELETE\
+		FROM \"$1\" e\
+		'WHERE "Report_Num" < (SELECT MAX("Report_Num")' FROM \"$1\" m\
+		'GROUP BY "Filer_ID", "From_Date"
+		HAVING  e."Filer_ID" = m."Filer_ID" AND e."From_Date" = m."From_Date");' \
```

Comment:
This is pretty hard to read because of the combination of SQL and bash.

Here is a more version that is easier for me to read at least, and also less confusing with all the escape shenanigans.
```bash
cat <<-QUERY | psql disclosure-backend
DELETE FROM "$1" outer
WHERE "Report_Num" < (
  SELECT MAX("Report_Num") FROM "$1" inner
  GROUP BY "Filer_ID", "From_Date"
  HAVING outer."Filer_ID" = inner."Filer_ID" AND outer."From_Date" = inner."From_Date"
);
QUERY
```

Also, I think it is equivalent for the inner query to use a WHERE instead of HAVING. Is that right?